pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2833,6 +2833,9 @@ type StorageScopeSpec struct {
 	// +nullable
 	// +optional
 	StorageClassDeviceSets []StorageClassDeviceSet `json:"storageClassDeviceSets,omitempty"`
+	// resize the osd crush weight once size of the osd pvc is increased
```

Comment:
Doc comments get rendered to rook.io now, so let's try to keep things capitalized and following good doc comment hygiene. 

And since this setting affects cluster rebalancing/stability, let's make sure we note that for users. I think this change might be helpful and still concise enough for users.

```suggestion
	// ResizeOsdCrushWeight allows Rook to resize the OSD CRUSH weight when the OSD PVC is increased. This allows cluster data to be rebalanced to make most effective use of new OSD space; however, data rebalancing can cause temporary cluster slowdown.
```

Reply:
Thanks this make more sense